components/browser/state/src/main/java/mozilla/components/browser/state/state/CustomTabSessionState.kt

```diff
@@ -0,0 +1,15 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.browser.state.state
+
+import java.util.UUID
+
+/**
+ * Value type that represents the state of a Custom Tab.
+ */
+data class CustomTabSessionState(
+    override val id: String = UUID.randomUUID().toString(),
```

Comment:
Could a default ID function be added to encapsulate this? `UUID.randomUUID().toString()` is used in a couple different places.

Reply:
Hmm, I think I would leave this as is. We're using UUIDs in various modules all over A-C. We won't gain much in terms readability or code reduction from moving this to `support-utils` (or similar). I'd also rather evaluate new situations and check if a UUID is really needed than having everyone just call `uniqueID` because it's convenient.